forge-gui/res/cardsfolder/upcoming/verdant_rejuvenation.txt

```diff
@@ -0,0 +1,6 @@
+Name:Verdant Rejuvenation
+ManaCost:6 G G
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Library | Destination$ Battlefield | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ X | ChangeType$ Creature,Enchantment,Planeswalker | SpellDescription$ Seek X creature, enchantment, and/or planeswalker cards, where X is the highest mana value among creatures you control. Put those cards onto the battlefield.
```

Comment:
this needs a `StackDescription$ SpellDescription`

we don't have a good stack description builder for seek stuff (which is fine)